base/uk.ac.stfc.isis.scriptgenerator.client/plugin_customization.ini

```diff
@@ -0,0 +1,68 @@
+uk.ac.stfc.isis.ibex.preferences/perspectives_not_shown=uk.ac.stfc.isis.ibex.client.e4.product.perspective.scriptGenerator
```

Comment:
We should not duplicate these preferences x2.

Either:
- Make the script generator refer to the other preference file
- Delete this file (I suspect most of the preferences are not required for the script generator)